system-variables.md

```diff
@@ -205,6 +211,18 @@ mysql> SELECT * FROM t1;
 >
 > Unlike in MySQL, the `max_execution_time` system variable currently works on all kinds of statements in TiDB, not only restricted to the `SELECT` statement. The precision of the timeout value is roughly 100ms. This means the statement might not be terminated in accurate milliseconds as you specify.
 
+### plugin_dir
```

Comment:
@morgo does setting these on a running server make sense? Looks like these should be only used during startup and remain read-only for the rest of the server lifetime.

Reply:
It makes sense. Details are correct, except I would call them "INSTANCE" scoped, not session since they have a getter function that always returns what is in config.